include/mrdox/Metadata/Javadoc.hpp

```diff
@@ -423,15 +519,18 @@ struct MRDOX_VISIBLE
     template<class T, class U>
     static
     void
-    append(AnyList<T>& list, AnyList<U>&& other) noexcept
+    append(doc::List<T>& list, doc::List<U>&& other) noexcept
     {
-        list.splice_back(std::move(other));
+        list.reserve(list.size() + other.size());
+        for(auto& p : other)
+            list.emplace_back(static_cast<T*>(p.release()));
```

Comment:
why not `std::move(p)`?

Reply:
because `T` might be a derived class of `U`